.github/workflows/config.yml

```diff
@@ -135,20 +141,23 @@ jobs:
             # Upload to the artifacts server of your choice.
             # Create a file with the service account data
             # use the bucket_upload script to upload your packs to google cloud storage
+            mkdir $NEW_PACKS_FOLDER
             python $GITHUB_WORKSPACE/repository/build_related_scripts/bucket_upload.py --service_account $GITHUB_WORKSPACE/service_account.json --packs_directory $NEW_PACKS_FOLDER --branch_name $BRANCH_NAME
             # Delete the service account file
             rm $GITHUB_WORKSPACE/service_account.json
 
             # 2. ========= UPLOAD DIRECTLY TO YOUR XSOAR MACHINE (WHEN MERGING TO MAIN REPO) =========
 
-            if [ $BRANCH_NAME != master ]; then
```

Comment:
It should upload only when it's merged into master that's the intended behavior so it should read `if [ $BRANCH_NAME == master ]`